src/partial_quicksync.rs

```diff
@@ -104,9 +104,19 @@ fn download_file(
     .extension()
     .is_some_and(|ext| ext == "zst")
     .then_some(".zst");
+  let version = env!("CARGO_PKG_VERSION");
   let url = format!("{}/{}", base_url, file_url(user_version, point, suffix));
+  let url_version = format!(
+    "{}/{}?version={}",
+    base_url,
+    file_url(user_version, point, suffix),
+    version
+  );
```

Comment:
Why do you have both `url` and `url_version`?
```suggestion
  let url = format!(
    "{}/{}?version={}",
    base_url,
    file_url(user_version, point, suffix),
    version
  );
```

+ fix code below

Reply:
url is printed then that's why created second version :)

Reply:
Including the query might be useful for copy-pasting the URL. However, if you don't want to print it, then how about creating a `url` with version and stripping everything after `?` for printing?

Reply:
Noone will copy & paste it with the quicksync version to use outside of quicksync, because why?

Reply:
I added the strip thingy.

Reply:
> Noone will copy & paste it with the quicksync version to use outside of quicksync, because why?

I did multiple times when testing it :shrug: